### PR TITLE
Backend votes

### PR DESCRIPTION
**Pull request by vincentfretin**
addTokenVote, deleteTokenVote, addGaugeVote, deleteGaugeVote, myVotes

### CODE REVIEW

**Comment by vincentfretin** [comment]
@cedricmessiant @maparent @ayyazdaniaryan @swergas 
Once this is merged, you will probably need to drop the test database and recreate it you have an error with vote_session drop when executing the tests.
```
PGPASSWORD=assembl_test dropdb --host localhost -U assembl_test assembl_test    
PGPASSWORD=assembl_test createdb --host localhost -U assembl_test assembl_test  
fab -c configs/testing.rc create_local_ini                                          
assembl-db-manage testing.ini bootstrap
```

File: assembl/alembic/versions/53af22ed84c1_votesession_widget.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import sqlalchemy as sa
+import transaction
+
+
+def upgrade(pyramid_env):
+    # Do stuff with the app's models here.
+    from assembl import models as m
+    db = m.get_session_maker()()
+    with transaction.manager:
+        # remove all existing vote sessions (there is no vote session currently in prod)
```
Comment (ayyazdaniaryan) [inline assembl/alembic/versions/53af22ed84c1_votesession_widget.py]: Ok fair enough.

File: assembl/graphql/vote_session.py [hunk 2]
```diff
@@ -168,6 +169,12 @@ def resolve_vote_spec_template_id(self, args, context, info):
     def resolve_vote_type(self, args, context, info):
         return self.type
 
+    def resolve_my_votes(self, args, context, info):
+        user_id = context.authenticated_userid
+        # use votes_of(user_id) instead of votes_of_current_user because
+        # request threadlocal is not properly set in tests
```
Comment (ayyazdaniaryan) [inline assembl/graphql/vote_session.py]: We have a fixture for this exactly like: `test_adminuser_webrequest`

Comment (vincentfretin) [reply assembl/graphql/vote_session.py]: graphql_request already uses test_adminuser_webrequest, but this doesn't push registry, request to the pyramid threadlocal manager for use with get_current_request()

**Pull request merged**